test/BaselineOutput/SingleDebug/SavePipe/PartitionedNamedDirectories-Schema.txt

```diff
@@ -1,5 +1,5 @@
 ---- PartitionedFileLoader ----
 3 columns:
-  L0: Text
-  Year: Text
-  Month: Text
+  L0: String
```

Comment:
I'll move them to Common folder after successful build run.
Otherwise it's gonna be messy PR.